tempesta_fw/t/functional/helpers/chains.py

```diff
@@ -197,6 +217,34 @@ def base(uri='/', method='GET', forward=True, date=None):
                                       server_response=backend_resp)
     return copy.copy(base_chain)
 
+def response_403(date=None, connection=None):
+    if date is None:
+        date = deproxy.HttpMessage.date_time_string()
+    if connection is None:
+        resp = deproxy.Response.create(status=403,
+                                       headers=['Content-Length: 0'],
+                                       date=date,
+                                       body='')
+    else:
+        resp = deproxy.Response.create(status=403,
```

Comment:
Copy-paste. It's better to use variable to store list of headers and add `Connection:` header if `connection is not None`, then call `deproxy.Response.create()` only once

Reply:
fixed

Reply:
fixed